### DISCUSSION TITLE
100% Font Opacity

### DISCUSSION THREAD

**Discussion by ArdonnieIV** [hikarin522/GlassIt-VSC#52] [Ideas]
Hey, this is by far my favorite vs-code extension!

The only thing I wish was different, is if there was an option to keep all text at 100% opacity.
This way you can make the backdrop completely transparent and still see the text if you have an appropriate wallpaper.

I don't know how the tool is currently implemented, and if it would be impossible given its current state.
But if possible, it would be a super nice addition.

**Comment by Oluvstrs** [comment]
oh, i think so ,too. keeping all text at 100% opacity is a really nice addtion.
